src/Auth/Authentication/Actions/EmailActivator.php

```diff
@@ -0,0 +1,122 @@
+<?php
```

Comment:
I don't think there's a reason to port this from Shield. It just makes a maintenance issue to keep it up to date to changes from Shield. Was there a valid reason to bring it forward?

Reply:
The reason to port this is that i include Themeable trait and made methods to use Auth config items to render templates